pkg/resources/user/userHelper.go

```diff
@@ -153,3 +155,51 @@ func GetIdentities(ctx context.Context, serverClient k8sclient.Client, user user
 	}
 	return identities, nil
 }
+
+func GetMultiTenantUsers(ctx context.Context, serverClient k8sclient.Client) (*usersv1.UserList, error) {
+
+	exclusionGroups = append(exclusionGroups, "dedicated-admin")
```

Comment:
Do we want to exclude some other groups: osd-sre-admins, cluster-admins, osd-sre-cluster-admins

Reply:
those are the current excluded groups https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/user/userHelper.go#L25 I am not sure we need to add more groups to this list, @philbrookes what do you think?